pkg/model/dev.go

```diff
@@ -176,6 +176,14 @@ func (dev *Dev) setDefaults() error {
 	if dev.Labels == nil {
 		dev.Labels = map[string]string{}
 	}
+	vMap := map[string]bool{}
```

Comment:
maybe extrapolate this to a `unique([]string)` function that returns the array without duplicates? we could use that in a few places.